backend/sozisel/lib/sozisel_web/schema/resolvers/event_resolvers.ex

```diff
@@ -1,11 +1,26 @@
 defmodule SoziselWeb.Schema.Resolvers.EventResolvers do
   alias SoziselWeb.Context
+  alias Sozisel.Repo
   alias Sozisel.Model.{Events, Sessions}
   alias Events.Event
   alias Sessions.Template
 
   import SoziselWeb.Schema.Middleware.ResourceAuthorization, only: [fetch_resource!: 2]
 
+  def get_event(_parent, %{id: event_id}, ctx) do
+    user_id = Context.current_user!(ctx).id
+
+    with %Event{} = event <- Repo.get(Event, event_id),
```

Comment:
Use bodyguard for that, moreover we have a resource middleware that can be used for auto fetching and checking what you did here by hand.

Anyway, who needed a separate query for a single event?

Reply:
@skusnierz requested that feature, because we have edition of the event, so I think that query single event is necessary